app/web/features/communities/newEvents/EventsList.tsx

```diff
@@ -0,0 +1,34 @@
+import { Typography } from "@material-ui/core";
+import { EventsType } from "features/queryKeys";
+import { useTranslation } from "i18n";
+import { COMMUNITIES, GLOBAL } from "i18n/namespaces";
+import mockEvents from "test/fixtures/events.json";
+
+import EventItem from "./EventItem";
+
+interface EventListProps {
+  eventType: EventsType;
+}
+
+const EventsList = ({ eventType }: EventListProps) => {
+  const { t } = useTranslation([GLOBAL, COMMUNITIES]);
+
+  const renderEventItems = () => {
+    return mockEvents.map((event) => (
+      <EventItem key={event.eventId} event={event} />
+    ));
+  };
+
+  return (
+    <div>
+      <Typography variant="h3">
```

Comment:
IMHO feels redundant

Reply:
It was in the design, do you prefer I remove it? It doesn't really take much performance to render it either way.